main.py

```diff
@@ -1082,6 +1085,25 @@ async def info(message: discord.Interaction):
         embedVar.set_footer(text="Last code update:")
     await message.followup.send(embed=embedVar)
 
+@bot.tree.command(description="View information about a cat")
+@discord.app_commands.rename(cat_type="type")
+@discord.app_commands.describe(cat_type="Select a cat type")
+@discord.app_commands.autocomplete(cat_type=cat_type_autocomplete)
+async def catinfo(message: discord.Interaction, cat_type: str):
```

Comment:
maybe a better name? this looks a bit ugly

Reply:
yeah I thought about the name for a bit; idk what to make it